src/window/window.rs

```diff
@@ -409,6 +410,16 @@ impl Window {
                            mem::transmute(&mut out[0]));
         }
     }
+    
+    /// Get the current screen as an image
+    pub fn snap_image(&self) -> image::ImageBuffer<image::Rgb<u8>, Vec<u8>> {
```

Comment:
Well actually free functions should keep the last module name (`imageops::flip_vertical`) so that it is clear where it comes from (i-e that is does not come from the current file).